shub/schedule.py

```diff
@@ -0,0 +1,41 @@
+from ConfigParser import NoSectionError, NoOptionError
+import click
+from click import ClickException
+from scrapinghub import Connection
+from shub import scrapycfg
+from shub.utils import find_api_key
+
+
+@click.command(help='Schedule a spider to run on Scrapy Cloud')
+@click.pass_context
+@click.argument("spider", type=click.STRING)
```

Comment:
Just a nitpick, but I think we should be consistent in the use of single or double quotes.